src/intl/en.json

```diff
@@ -153,6 +153,12 @@
       "server_invalid_status": "Error fetching catalyst statuses"
     },
     "grant": {
+      "funding": {
+        "invalid": "Funding is invalid",
+        "too_low": "Funding is too low. Minimum is 100 USD",
```

Comment:
this numbers could ve a variable, like:  `t('funding.too_low', {value: MIN_CONSTANT})`